release-centos7-llvm/dockerfiles/misc/bake_llvm_base_amd64.sh

```diff
@@ -33,7 +33,7 @@ function bake_llvm_base_amd64() {
 
     # LLVM
     source $SCRIPTPATH/bootstrap_llvm.sh
-    bootstrap_llvm "13.0.0"
```

Comment:
Could we use (latest minor fix version) LLVM 15.0.7? It should contain some useful bug fixes.

Reply:
yes, updated.